health_check.md

```diff
@@ -0,0 +1,88 @@
+# Etcd Cluster Health Check
+
+##  Synopsis
```

Comment:
I think we could put `Synopsis` to the last, since it's normal to explain what is it first and then tell people how to use it.

Reply:
update